libraries/parcelable/src/main/java/com/uber/okbuck/example/common/Person.java

```diff
@@ -1,8 +1,11 @@
 package com.uber.okbuck.example.common;
 
+import android.content.Context;
 import android.os.Parcel;
 import android.os.Parcelable;
 
+import com.uber.okbuck.example.parcelable.R;
```

Comment:
@kageiit test updated. `com.uber.okbuck.example.common.R` works, but `com.uber.okbuck.example.parcelable.R` does not